pkg/codegen/schema/mock_pulumi_schema.go

```diff
@@ -22,11 +22,20 @@ func newPulumiPackageReference() *Package {
 				},
 			},
 		},
+		Provider: ResourceSpec{
+			InputProperties: map[string]PropertySpec{
```

Comment:
We should probably add a description here, since this stuff is exposed via the LSP server.